posts/2023-09-26-23.0.0.10-beta.adoc

```diff
@@ -0,0 +1,628 @@
+alpha_timer_seconds_bucket{scope="application",le="0.1",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="0.2",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="1.0",} 1.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="+Inf",} 2.0  <2><3>
+alpha_timer_seconds_count{scope="application",} 2.0
+alpha_timer_seconds_sum{scope="application",} 6.333
+----
+
+* The Prometheus metric type is `histogram`. Both the quantiles or percentiles and buckets are represented under this type. 
+* The `le` tag represents _less than_ and is for the defined buckets, which are converted to seconds.
+* Prometheus requires that a `+Inf` bucket, which counts all hits.
+    
+  
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26183
+// Contact/Reviewer: yasmin-aumeeruddy
+// // // // // // // // 
+[#mptel]
+== MicroProfile Telemetry 1.1
+
+MicroProfile Telemetry 1.1 provides developers with the latest Open Telemetry technology; the feature now consumes OpenTelemetry-1.29.0, updated from 1.19.0. Consequently, a lot of the dependencies are now stable. 
+
+To enable the MicroProfile Telemetry 1.1 feature, add the following configuration to your `server.xml`:
+
+[source,xml]
+----
+<features>
+   <feature>mpTelemetry-1.1</feature>
+</features>
+----
+
+Additionally, third-party APIs must be made visible for your application in the `server.xml`:
+
+[source,xml]
+----
+<webApplication location="demo-microprofile-telemetry-inventory.war" contextRoot="/">
+    <!-- enable visibility to third party apis -->
+    <classloader apiTypeVisibility="+third-party"/>
+</webApplication>
+----
+
+For more information about MicroProfile Telemetry, see:
+
+* link:https://github.com/eclipse/microprofile-telemetry[MicroProfile Telemetry spec]
+* link:https://github.com/open-telemetry/opentelemetry-specification/blob/v1.11.0/specification/trace/api.md[OpenTelemetry spec]
+* link:{url-prefix}/docs/latest/microprofile-telemetry.html[MicroProfile Telemetry docs]
+   
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26222
+// Contact/Reviewer: Azquelt
+// // // // // // // // 
+
+[#mpapi]
+== MicroProfile OpenAPI  endpoint path configuration
+
+MicroProfile OpenAPI generates and serves OpenAPI documentation for JAX-RS applications that are deployed to the Open Liberty server. The OpenAPI documentation is served from `/openapi` and a user interface for browsing this documentation is served from `/openapi/ui`.
+
+With MicroProfile OpenAPI 3.1, you can configure the paths for these endpoints by adding configuration to your `server.xml`. For example:
+     
+[source,xml]
+----
+<mpOpenAPI docPath="/myOpenAPI/path" uiPath="/myUi/Path" />
+----

```

Comment:
Could I make a small revision here to make it clearer what the config example does please?

```suggestion
With MicroProfile OpenAPI 3.1, you can configure the paths for these endpoints by adding configuration to your `server.xml`. For example:
     
[source,xml]
----
<mpOpenAPI docPath="/my/openapi/doc/path" uiPath="/docsUi" />
----

Setting this configuration on a local test server would mean that you would then access the OpenAPI document at `localhost:9080/my/openapi/doc/path` and the UI at `localhost:9080/docsUi`.
```